config-provisioning/src/main/resources/configdefinitions/config-server-filter.def

```diff
@@ -0,0 +1,9 @@
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Consider renaming the file to `config-server-security.def`. This config is used by classes that are not filters.